builds/azure-pipelines/template-steps-build-test.yml

```diff
@@ -19,10 +19,15 @@ steps:
     packageType: sdk
     version: '3.1.x'
 
-- script: npm install -g azure-functions-core-tools
+- task: npmAuthenticate@0
```

Comment:
Make sure you check with a buddy build that these are pulling from the right npmrc file in our ADO pipelines.

Reply:
yes, am going to test with a buddy build.